### PR TITLE
Adding PR template, copying the one Dan added for Android

### PR DESCRIPTION
**Pull request by darinwf**

### CODE REVIEW

**Comment by darinwf** [comment]
@dan-alcantara-neeva FYI

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+### Screenshots and screen recordings
+Images or videos showing the effect of your PR.  Examples include:
+* For new Views, add images of `_Previews` you have added for both light
+  and dark mode
+* Before and after screenshots showing the effect of your styling changes
+* Videos showing you performing the flow that you have changed
+
+## Issues addressed
+* Link(s) to Github issue(s) (if applicable)
```
Comment (EdwardJTL) [inline .github/pull_request_template.md:28]: do we have a bot that will close those issues after?

Comment (dan-alcantara-neeva) [reply .github/pull_request_template.md:28]: Technically Github should do it for you if you use one of the linked keywords:
https://docs.github.com/en/enterprise-server@3.4/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests

Comment (EdwardJTL) [reply .github/pull_request_template.md:28]: ah! makes sense 😅 totally forgot to those existed

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+or attempting to add new functionality_
+
+* More details about the changes
+* Even more details about the changes
+* Yet more details about the changes
+
+## Checklists
+
+### How was this tested?
+[ ] I tested this manually
```
Comment (evancrow) [inline .github/pull_request_template.md:12]: Should have a `-` before each `[ ]`, that way it makes an actual check box. @dan-alcantara-neeva we should also make this change for the Android version as well.

- [ ] <- so it looks like this

Comment (dan-alcantara-neeva) [reply .github/pull_request_template.md:12]: Yeah, missed that when I was looking at how the other repo did it.  Will add it on our side.

**Review by evancrow — APPROVED** [review]

**Pull request merged**